WORKSPACE

```diff
@@ -30,6 +30,10 @@ google_repos()
 load("//third_party/google:deps.bzl", "google_deps")
 google_deps()
 
+# cboer reader & writer from open-dice
+load("//third_party/open-dice:repos.bzl", "open_dice_repos")
+open_dice_repos()
+
```

Comment:
There is no specific problem with loading this here, but the early ordering of some deps can require some care.  As a general utility library, I'd suggest loading this later (certainly after the compilers and other language support).  I'd suggest somewhere after line 116 (FreeRTOS, etc).